app/packages/state/src/recoil/pathData/counts.ts

```diff
@@ -67,9 +67,10 @@ export const count = selectorFamily({
         }
 
         if (first === "_label_tags" && value) {
-          return get(cumulativeCounts({ ...params, ...MATCH_LABEL_TAGS }))[
-            value
-          ];
+          return (
+            get(cumulativeCounts({ ...params, ...MATCH_LABEL_TAGS }))[value] ??
```

Comment:
Is it possible for the get to return none and then throw an error when trying to access value from none?

Reply:
This is a small detail I fixed when checking the legacy sidebar mode. We want zero to return here as it does on line 95. Currently "of X" is shown instead of "0 of X". See screenshot.
<img width="1470" alt="Screenshot 2025-04-01 at 10 40 17 AM" src="https://github.com/user-attachments/assets/6f7e6d69-3bdc-49ad-a4f9-d845ce07f2ec" />